src/Sulu/Bundle/PageBundle/Search/EventListener/PermissionListener.php

```diff
@@ -37,6 +38,11 @@ public function __construct(DocumentManagerInterface $documentManager, SearchMan
         $this->searchManager = $searchManager;
     }
 
+    public static function getSubscribedEvents(): array
+    {
+        return ['sulu_security.permission_update' => 'onPermissionUpdate'];
```

Comment:
isnt there a constant for this?

Reply:
Good catch didn't found first as the constant was in `Sulu\Component` namespace 🙈